CMakeLists.txt

```diff
@@ -580,7 +612,7 @@ if(COMPILE_CPU)
         endif(CBLAS_FOUND)
       endif(BLAS_FOUND)
     endif(MKL_FOUND)
-  endif(USE_WASM_COMPATIBLE_BLAS)
+  endif(USE_ONNX_SGEMM)
```

Comment:
This renaming is fine.  However, you shouldn't be using ONNX_SGEMM at all on ARM.